README.md

```diff
@@ -6,8 +6,16 @@ Base repository for the [Core Data Structures](http://jsdev.learnersguild.org/go
 
 ## Installation and Setup
 
-_Fill this out_
+For this project, we created both tests and implementations of core data structures. Begin with installing your dependencies from the package.json file.
 
```

Comment:
In the README Make sure that you include the Specs and Quality Rubric so as to give your app and it's contributors accountable goals to overcome.